dataland-e2etests/src/test/kotlin/org/dataland/e2etests/customApiControllers/CustomVsmeDataControllerApi.kt

```diff
@@ -71,7 +72,8 @@ class CustomVsmeDataControllerApi(private val token: String) {
                 documents,
             ),
         ).execute()
-        require(response.isSuccessful) { "Request failed, response is: $response" }
+        if (response.code == 403) { throw ClientException("Client error : 403 ") }
```

Comment:
Out of curiosity, why do we throw an exception with a manual message here and do not use the specific subclass?

Reply:
Diesen client in diesem File musste ich selbst händisch schreiben, weil der autgenerierte client von openApi leider keinen vernünftigen multipart request machen kann.

An dieser Stelle hier möchte ich, dass mein custom client sich genauso verhält wie der autogenerierte client von openApi im Falle eines 403 errors.
Grund: Dann kann ich in den Tests auf dieselbe error message asserten wie bei den autogenerierten clients.

Daher "ahme" ich hier einfach das nach, was die autogenerierten clients auch machen.